thefuck/rules/conda_mistype.py

```diff
@@ -0,0 +1,17 @@
+import re
+from thefuck.utils import replace_command, for_app
+
+
+@for_app("conda")
+def match(command):
+    """
+    Match a mistyped command
+    """
+    return "conda" in command.script and "Did you mean 'conda" in command.output
```

Comment:
Once the function is anotated with `@for_app("conda")` there's no need to check if it is included in `command.script`.
```suggestion
    return "Did you mean 'conda" in command.output
```